assets/base/i18n/en.json

```diff
@@ -1,558 +1,558 @@
 {
```

Comment:
You are changing the format of the whole file, you should respect that.

Reply:
👍